src/main/java/uk/gov/hmcts/reform/sscs/config/TribunalsEventQueueListenerConfig.java

```diff
@@ -2,45 +2,40 @@
 
 import com.azure.core.amqp.AmqpRetryMode;
 import com.azure.core.amqp.AmqpRetryOptions;
```

Comment:
Why was this class changed? This ticket concerns HMC -> Hearings API not Tribs -> Hearings API